src/validator.ts

```diff
@@ -110,6 +112,66 @@ export function addPseudoValue(parameter: string, value: string){
     }
 };
 
+export function addCustomResourceAttributeValue(resource: string, attribute: string, value: any){
+    let attrType = determineValueType(value);
+    let oldSpec = {};
+    let newSpec = {
+      'Attributes': {
+        [attribute]: {
+          PrimitiveType: attrType,
+          Value: value
+        }
+      }
+    };
+
+    // extend defintion for existing resource type
+    if (RegExp(/::/).test(resource)) {
```

Comment:
Can we use indexof() for this?

Reply:
I don't see why not ...

Reply:
https://xkcd.com/1171/ ;)

`str.indexOf("::") > -1` ?

I dunno, @akdor1154 any suggestions on the neatest/cleanest way of simple substring checking in TS?

Reply:
:)